assets/admin/package.json

```diff
@@ -60,6 +60,7 @@
         "webpack-manifest-plugin": "^2.0.2"
     },
     "engines": {
-        "npm": "<7"
+        "node": ">=12",
```

Comment:
Node 10 did not work before because the `@ckeditor/ckeditor5-core` is not compatible with `Node 10`. see https://github.com/sulu/skeleton/pull/102#discussion_r604864429